static/js/narrow.js

```diff
@@ -555,6 +555,10 @@ function pick_empty_narrow_banner() {
         }
     } else if ((first_operator === "stream") && !stream_data.is_subscribed(first_operand)) {
         // You are narrowed to a stream to which you aren't subscribed.
+        var sub = stream_data.get_sub(narrow.stream());
```

Comment:
We might be able to simplify this to `... && !stream_data.get_sub(narrow.stream())) {`